Documentation.docc/V4_API_Migration_guide.md

```diff
@@ -110,7 +110,7 @@ These types replace native StoreKit types in all public API methods that used th
 | RCAttributionNetwork | ``AttributionNetwork`` |
 | RCIntroEligibility | ``IntroEligibility`` |
 | RCIntroEligibilityStatus | ``IntroEligibilityStatus`` |
-| RCPaymentMode | ``StoreProductDiscount.PaymentMode`` |
+| RCPaymentMode | ``StoreProductDiscount/PaymentMode-swift.enum`` |
```

Comment:
Not sure why this is correct, but Xcode autocompleted it and as you can see in the screenshot it works.
`StoreProductDiscount/PaymentMode` doesn't work.

Reply:
It's documented here https://developer.apple.com/documentation/xcode/formatting-your-documentation-content/#Link-to-Symbols-and-Other-Content it's because `StoreProductDiscount/PaymentMode` is ambiguous, between the `PaymentMode` enum and the `paymentMode` variable (it's case-insensitive)